monkestation/code/datums/keybinding.dm

```diff
@@ -0,0 +1,14 @@
+/datum/keybinding/human/equip_swap
+	key = "Ctrl+E"
```

Comment:
Isn't ctrl+E also bound to another thing?

Reply:
Shift E is but not Ctrl E